### DISCUSSION TITLE
HALMET can't connect to Raspberry pi 4 Hotspot

### DISCUSSION THREAD

**Discussion by Stonepaiste** [hatlabs/discussions#97] [Q&A]
I'm having a problem with my Halmet. It won't connect to my raspberry pi 4 Hotspot. The raspberry is configured to a 2.4 Ghz wifi with a WPA/WPA2 password. it tries to connect only to disconnect right after. Do anyone have a solution to this. ?

**Comment by big-col** [comment]
Are you still having this issue? I'm happy to try and replicate if you are

**Reply by Stonepaiste** [reply]
No. I’m not really sure what worked but i think after hitting the “reset” button on the halmet configure popup panel it did something.. Don’t know why that would help since i tried wiping the flash multiple times on the hamlet. But after trying everything else 50 times i hit it and it worked. Probably should have started with that. But thanks for getting back.
